Unix/installbuilder/conf/omiserver.conf

```diff
@@ -46,3 +46,14 @@ keyfile=/etc/opt/omi/ssl/omikey.pem
 configfile=/etc/opt/omi/conf/omiserver.conf
 #NoSSLv2=true
 #NoSSLv3=false
+# To enable NTLM authentication via negotiation (SPNEGO), specify the location 
+# of the credentials file. This file is in the form
+#
+# <domain>|<username>:<password> 
+#
+# For further information, take a look at the file Unix/doc/setup-ntlm-omi.md
+#
+#
+#NtlmCredsFile=PATH
+# For example
+NtlmCredsFile=/etc/opt/omi/.creds/ntlm
```

Comment:
Please move this segment to a slightly more obvious block, like just after 22, putting a blank line after this segment (to separate from other segments).